client/js/socket.js

```diff
@@ -40,7 +40,7 @@ function handleDisconnect(data) {
 	const message = data.message || data;
 
 	$("#loading-page-message, #connection-error").text(`Waiting to reconnect… (${message})`).addClass("shown");
-	$(".show-more-button, #input").prop("disabled", true);
+	$(".show-more button, #input").prop("disabled", true);
```

Comment:
I targeted `button` on purpose here because I was aiming for the button, not its styling. I can change if you prefer, but I think it makes more sense that way.